doc/source/conf.py

```diff
@@ -42,6 +42,7 @@
 )
 
 # Use the default pyansys logo
+html_baseurl = f"https://{cname}/version/stable"
```

Comment:
To add the canonical tag to the rendered html

Reply:
We have to document this...

Reply:
This is no longer required since the action overwrites any canonical links. I imposed this to not affect any of the `conf.py` files from our projects.

I will add the documentation for this.